src/dbus_api/filesystem/filesystem_3_7/props.rs

```diff
@@ -0,0 +1,18 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+use dbus::arg::IterAppend;
+use dbus_tree::{MTSync, MethodErr, PropInfo};
+
+use crate::dbus_api::{
+    filesystem::shared::{self, get_filesystem_property},
+    types::TData,
+};
+
+pub fn get_fs_origin(
+    i: &mut IterAppend<'_>,
+    p: &PropInfo<'_, MTSync<TData>, TData>,
+) -> Result<(), MethodErr> {
+    get_filesystem_property(i, p, |(_, _, f)| Ok(shared::fs_size_limit_prop(f)))
```

Comment:
Should this be ``shared::fs_origin_prop(f)``?

When no origin is set for a filesystem does this return ``(false, "")``?